### DISCUSSION TITLE
Is there any way I can use System Informer on Windows Insider builds?

### DISCUSSION THREAD

**Discussion by Aholicknight** [winsiderss/systeminformer#1794] [Questions / Answers] [answered]
I have been using System Informer with kernel mode enabled for some time now on my Windows 11 machine, but as soon as I installed the latest Windows 11 Insider build (beta branch) the kernel mode driver stopped working. Is there anyway to _force_ the drive to load, or do I have to switch back to the non-insider build? Thank you.

**Comment by jxy-s** [comment, marked as answer]
The kernel driver doesn't support insider builds, for now. We have plans to try to support them in the future once our tooling improves. There is no ETA for this work.

Previously the driver would "silently hide" the fact that it didn't fully support insider builds. With the advent of the newer driver there is now a requirement that we have the internal offsets in place in order to load the driver and ensure the integrity of the client. Due to this requirement the driver will fail to load if the offsets (dynamic data) aren't present. Fully supporting insider builds introduces a challenge to keep up with the more frequent Windows Insider release cadence. Once our tooling/automation improves we might be able to support them.
